heliopy/data/util.py

```diff
@@ -35,7 +35,7 @@ def process(dirs, fnames, extension, local_base_dir, remote_base_url,
         same length as *dirs*.
     extension : str
         File extension of the raw files. **Must include leading dot**.
-    local_base_dir : str
+    local_base_dir : PosixPath
```

Comment:
Can this be a string or a `PosixPath`? Can then be converted to a Path using `local_base_dir = Path(local_base_dir)` inside the function.